### PR TITLE
Add Obj-C and Swift to Quickstart Guide.

### PR DESCRIPTION
**Pull request by tcard**

### CODE REVIEW

File: content/root/quick-start-guide.textile [hunk 1]
```diff
@@ -14,6 +14,28 @@ When using Ably in a browser, simply include the Ably Javascript library within
...
+  ```
+
+blang[swift].
+  Add this line to your application's Podfile:
+
+  bc. pod 'Ably', '~> 0.8'
+
+  And install with @pod install@. Then in your files: 
+
+  ```[objc]
```
Comment (mattheworiordan) [inline content/root/quick-start-guide.textile]: isn't this supposed to `swift`?

Comment (mattheworiordan) [inline content/root/quick-start-guide.textile]: Also, if you read the text at https://github.com/ably/docs/pull/94/files#diff-f9d1ed36577e92a45ffb881113c2205fL13, you'll see that you need to perhaps change that copy to be included in all languages other than the ones you've suggested

File: content/root/quick-start-guide.textile [hunk 2]
```diff
@@ -41,10 +63,15 @@ ably.connection.on(:connected) do
 end
 
 bc[swift]. let ably = ARTRealtime(key: "{{API_KEY}}")
-ably.connection.on(:connected) do
-  print("That was simple, you're now connected to Ably in realtime")
+ably.connection.on(.Connected) {
+    print("That was simple, you're now connected to Ably in realtime")
```
Comment (mattheworiordan) [inline content/root/quick-start-guide.textile]: Thanks, I had written that before, so must have lost it in a copy & paste, thanks for picking that up

**Comment by mattheworiordan** [comment]
One or two comments, but LGTM

File: content/root/quick-start-guide.textile (hunk 1 above)
Comment (tcard) [inline content/root/quick-start-guide.textile]: > isn't this supposed to swift?

Fixed at 2661702.

> you'll see that you need to perhaps change that copy to be included in all languages other than the ones you've suggested

I don't know how this should be done for all languages. I realize that the browser copy should be only for JavaScript, but shouldn't that be in a different pull request?

**Comment by mattheworiordan** [comment]
@tcard I have updated the PR, tell me if you're happy with the changes

**Comment by tcard** [comment]
LGTM, one small comment.
